lib/screens_web/controllers/v2/screen_api_controller.ex

```diff
@@ -16,10 +16,11 @@ defmodule ScreensWeb.V2.ScreenApiController do
     end
   end
 
-  def show(conn, %{"id" => screen_id, "last_refresh" => last_refresh}) do
+  def show(%{params: params} = conn, %{"id" => screen_id, "last_refresh" => last_refresh}) do
     is_screen = ScreensWeb.UserAgent.is_screen_conn?(conn, screen_id)
+    screen_side = get_screen_side(params)
```

Comment:
A helper function is fine, but if you want you could also use the [`data[key]` syntax](https://hexdocs.pm/elixir/Access.html) or `Map.get`, which both return `nil` if the key doesn't exist.
```suggestion
    screen_side = params["screen_side"]
```
```suggestion
    screen_side = Map.get(params, "screen_side")
```